app/models/test_analytics_json.rb

```diff
@@ -0,0 +1,5 @@
+TEST_ANALYTICS_JSON = {
```

Comment:
Combining these does make it less obvious which table is generated from where when you're editing the source, so I'd be tempted to split these out.